### PR TITLE
Add variable to merge profile variable and vms variable

### PR DESCRIPTION
**Pull request by RouxAntoine**
currently cloud-init variables at vm level are prioritized over the cloud-init variables of profiles
same for sysprep variable. 

To relaxed this behavior we create new property merge_with_profile which allow to combine two level. cf exemple into [readme](./README.md)

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: tasks/manage_state.yml
Comment (mnecas) [inline tasks/manage_state.yml]: please remove the `== True`

File: tasks/manage_state.yml
Comment (mnecas) [inline tasks/manage_state.yml]: please remove the ` == False` and use `not`

File: tasks/manage_state.yml
Comment (mnecas) [inline tasks/manage_state.yml]: please remove the ` == False` and use `not`

File: tasks/manage_state.yml
Comment (mnecas) [inline tasks/manage_state.yml]: please remove the ` == True`

File: README.md
Comment (mnecas) [inline README.md]: Would rename the variable. People who did not read the docs would misunderstand it.
How about `init_merge`? Opened for any other suggestion.
@jan-zmeskal @dangel101 any ideas?

Comment (mnecas) [reply README.md]: Instead of saying that it is undefined, but default False, please add the variable to the defaults/main.yml.
Also, formulate the description like https://github.com/oVirt/ovirt-ansible-vm-infra/pull/129/files#diff-04c6e90faac2675aa89e2176d2eec7d8R68

**Review by mnecas — CHANGES_REQUESTED** [review]
Thank you for the PR really interesting idea!
Got some comments please check it out.

**Comment by mnecas** [comment]
cc @mwperina

**Comment by mnecas** [comment]
ci test please

File: tasks/manage_state.yml
Comment (RouxAntoine) [reply tasks/manage_state.yml]: remove useless `==True` into commit ddbdb8a

File: tasks/manage_state.yml
Comment (RouxAntoine) [reply tasks/manage_state.yml]: done into commit ddbdb8a

File: tasks/manage_state.yml
Comment (RouxAntoine) [reply tasks/manage_state.yml]: done into commit ddbdb8a

File: README.md
Comment (RouxAntoine) [reply README.md]: thanks again for fast review 😃 

I set default value for merge_with_profile into defaults/main.yml, but i wait final decision to change name. 

I don't understand what means prefix `init_` ? Maybe for `cloud_init` ? but we also support merging of `sysprep` property 🤔

File: tasks/manage_state.yml
Comment (RouxAntoine) [reply tasks/manage_state.yml]: remove useless ==True into commit ddbdb8a

File: README.md
Comment (RouxAntoine) [reply README.md]: WDYT about init_merge for cloud_init and sysprep_merge for sysprep ?

**Comment by mwperina** [comment]
Development moved to https://github.com/ovirt/ovirt-ansible-collection If this is still relevant, please reopen there.

**Pull request closed without merging**
